### PR TITLE
Introducing Persisters

### PR DESCRIPTION
**Pull request by mllocs**
First step on creating resources as side effects. In the future, when creating/updating a Post/Member/Transfer, it will create Events as side effects. Events will be used to create a kind of "Activity thread" section in a Time Bank. 

About the naming: https://english.stackexchange.com/questions/206893/persister-or-persistor

### CODE REVIEW

**Comment by mllocs** [comment]
👧 👧

File: spec/services/persister/post_persister_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+describe Persister::PostPersister do
+  let(:organization) { Fabricate(:organization) }
+  let(:user) { Fabricate(:user) }
+  let(:category) { Fabricate(:category) }
+  let(:post) { Fabricate(:post, organization: organization) }
+
+  describe '#save' do
+    it 'saves the post' do
+      post = Offer.new(organization: organization, user: user, category: category, title: 'Title')
+      persister = ::Persister::PostPersister.new(post)
```
Comment (enricostano) [inline spec/services/persister/post_persister_spec.rb]: Move both to `let` and reuse in examples

File: spec/services/persister/post_persister_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+      persister.save
+
+      expect(post).to be_persisted
+    end
+  end
+
+  describe '#update_attributes' do
+    it 'updates the attributes' do
+      persister = ::Persister::PostPersister.new(post)
```
Comment (enricostano) [inline spec/services/persister/post_persister_spec.rb]: Same

File: app/models/user.rb [hunk 3]
```diff
@@ -61,10 +61,18 @@ def user
   end
 
   def add_to_organization(organization)
-    organization && members.
-      find_or_create_by(organization: organization) do |member|
-      member.entry_date = DateTime.now.utc
-    end
+    return unless organization
+
+    member = members.where(organization: organization).first_or_initialize
```
Comment (enricostano) [inline app/models/user.rb]: `members` or `Member`?

Comment (mllocs) [reply app/models/user.rb]: `members` would be the same as doing `Member.where(user: self, ...`

**Pull request merged**